packages/client/modules/userDashboard/components/Organization/OrganizationPage.tsx

```diff
@@ -36,7 +36,7 @@ const OrganizationPage = (props: Props) => {
           <Route
             exact
             path={`${match.url}`}
-            render={(p) => <OrgBilling {...p} organization={organization} />}
+            render={() => <Redirect to={`${match.url}/${BILLING_PAGE}`} />}
```

Comment:
As `/me/organizations/${orgId}` defaults to Billing, we may as well redirect them to the billing page. This makes highlighting the active left nav item easier